game/src/test/scala/io/aigar/game/ResourcesSpec.scala

```diff
@@ -0,0 +1,15 @@
+package io.aigar.game
+
+import org.scalatest._
+
+class ResourcesSpec extends FlatSpec with Matchers {
+  "Resources" should "spawn at the right quantity" in {
+    val resources = new Resources
+
+    val state = resources.state
+
+    state.regular.size should equal(Resources.MaxRegular)
```

Comment:
Minor, you can leave it like that, but cool to know that [`ScalaTest` has](http://www.scalatest.org/user_guide/using_matchers#javaCollectionsAndMaps) `state.regular should have size Resources.MaxRegular`.